app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/actions/AuthenticatedActionWithRetrievedData.scala

```diff
@@ -154,7 +155,7 @@ class AuthenticatedActionWithRetrievedData @Inject()(
   // work out if it is an organisation or not
     enrolments.getEnrolment("HMRC-TERS-ORG") match {
       case None =>
-        Left(SeeOther(routes.RegisterTrustController.registerYourTrust().url))
+        Right(AuthenticatedRequestWithRetrievedData(UserType.NonTrustOrgansation, request))
 
```

Comment:
Shouldn't UserType.NonTrustOrganisation change to UserType.OrganisationUnregisteredTrust?

Reply:
yup, renamed it